rtc/getting-started.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Getting started with RTC
+title_nav: Getting started
+description: Getting started with RTC
+keywords: rtc
+---
+
+{% assign beta_feature = "The RTC plugin" %}
+{% assign pre-release_type = "Closed Beta" %}
+{% include misc/beta-note.md %}
+{{site.requires_5_6v}}<br>
+{{site.premiumplugin}}<br>
+
+## Prerequisites
+
+* A {{site.cloudname}} API key from [{{site.accountpage}}]({{site.accountsignup}}).
+* A JWT key generated at [{{site.accountpageurl}}]({{site.accountpageurl}}). Temporarily store the generated keys for use in the following procedure.
+
+## Basic RTC setup
+
+The following example shows how to get the RTC plugin up and running using hard coded values. For a production setup, the document ID and secret would be retrieved from your server. This examples also assumes that a JWT provider endpoint exists at '/jwt'. For information on setting up a JWT endpoint, see: [JWT authentication]({{site.baseurl}}/rtc/jwt-authentication/).
+
+```js
+const yourDocumentId = 'some-document-id';
+const yourSecretKey = 'your secret 🔑';
```

Comment:
Is it wise to use a unicode emoji here? This may not render well on some devices.
```suggestion
const yourSecretKey = 'your secret key';
```

Reply:
no https://www.tiny.cloud/docs/configure/contributing-docs/#glossary